api/external/infra_proxy/response/policyfiles.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+package chef.automate.api.infra_proxy.response;
+option go_package = "github.com/chef/automate/api/external/infra_proxy/response";
+
+message Policyfiles {
+    // List of policy files.
+    repeated PolicyfileListItem policies = 1;
+}
+
+message PolicyfileListItem {
+    // Name of the policy file.
+    string name = 1;
+    // Revision ID of the policy file.
+    string revision_id = 3;
+    // Policy group of the policy file.
+    string policy_group = 2;
+}
+
+message Policyfile {
+    // Name of the policy file.
+    string name = 1;
+    // Name of the policy group.
+    string policy_group = 2;
+    // Revision Id of the policy.
+    string revision_id = 3;
+    // Run list associated with the policy.
+    repeated string run_list = 4;
+    // Named the run list associated with the policy.
```

Comment:
In policyfile, we have provision to provide the name to the group of policy

eg:

```
"named_run_lists": {
    "update_app": [
      "recipe[learn_chef_apache2::default]",
	  "recipe[starter::default]"
    ]
  },

```

Reply:
I understand that it is a thing with a name. But those comments in the proto file go into generated docs for the user. Does your comment on L29 help the user understand that field in the response? It does not for me, but I am not familiar with policyfiles. But my original question was really does it provide enough info for someone who is familiar with policyfiles. I do not think so, but I am not necessarily making an informed opinion.

Reply:
I do agree with your point, this might be we can have better comments. I am okay with suggestions here?

Here what policyfile docs say about `named_run_lists`:


```
named_run_list "NAME", "ITEM1", "ITEM2", ...
```

Specify a named run-list to be used as an alternative to the override run-list. This setting should be used carefully and for specific use cases, like running a small set of recipes to quickly converge configuration for a single application on a host or for one-time setup tasks. For example:

```
named_run_list :update_app, "my_app_cookbook::default"
```
Ref: https://docs.chef.io/policyfile/#settings

Reply:
@vsingh-msys is going to make an issue to come back address this concern so we can keep moving on the work, as we want to loop a few opinions into the resolution